src/CommunityToolkit.Maui/Behaviors/Validators/TextValidationBehavior.shared.cs

```diff
@@ -7,6 +8,24 @@
 
 namespace CommunityToolkit.Maui.Behaviors;
 
+/// <summary>Flags to indicate what treatment <see cref="ValidationBehavior.Value"/> should receive prior to validation with <see cref="ValidationBehavior"/> or subclasses. This can be used to trim or ignore whitespace for instance. This value might be ignored by a behavior if <see cref="ValidationBehavior.Value"/> isn't of type <see cref="string"/>.</summary>
+[Flags]
+public enum TextDecorationFlags
```

Comment:
Hey Vlad! I started the process of creating an `Enums` folder and moving all of the enums to it. 

But then I quickly realized that this would require a new `xmlns`, making life harder for XAML devs.

I agree that we should move the files, but I think we should wait until #144 is completed to avoid making life hard for XAML devs before #144 makes it simpler.

I've added this task to a Proposal #219, which we'll complete after #144 👍